EIPS/eip-7727.md

```diff
@@ -0,0 +1,228 @@
+---
+eip: 7727
+title: EVM Transaction Bundles
+description: Enable EVM support for transaction bundles without revert protections.
+author: Lily Johnson (@lilyjjo)
+discussions-to: https://ethereum-magicians.org/t/eip-7727-evm-transaction-bundles/20322
+status: Draft
+type: Standards Track
+category: Core
+created: 2024-06-24
+requires: 2718
+---
+
+## Abstract
+
+This EIP introduces two new [EIP-2718](./eip-2718.md) transaction types and one new opcode, enabling smart contracts and transactions to delegate their local sequencing rights to an off-chain entity. These new transaction types work together to create EVM-native 'bundles', which are similar but weaker than the PBS bundles offered by builders to searchers.
+
+One of the EIP-2718 transactions is an extended normal transaction that supports:
+
+- Specifying the entity who can place the transaction in a bundle.
+- An optional block number that the transaction is valid in.
+
+The other EIP-2718 transaction is a ‘meta’ transaction that does not enter into execution, but rather orders transactions of its own type and the other new type. The 'meta' transaction can also specify an entity allowed to include it in a bundle and a valid block number.
+
+The new opcode reveals to the EVM the entity that placed the transaction in a bundle if the transaction was in a bundle.
+
+## Motivation
+
+Currently, a single block builder has unrestricted control over the final sequencing of a block’s transactions. This poses a problem, as sequencing—the choice of who gets to interact with specific pieces of state and in what order—significantly influences value flow. The objective of this EIP is to allow more parties to participate in the construction of single blocks by exposing sequencing concepts inside of the EVM. This change would enable EVM applications to reclaim some of the sequencing value that is currently monopolized by block builders, redirecting it back to the applications themselves.
+
+## Specification
+
+### Constants
+
+| Name | Value |
+| --- | --- |
+| FORK_BLOCK | TBD |
```

Comment:
```suggestion
| FORK_BLOCK | TBD <!-- TODO --> |
```

If you use HTML-style comments for your TBDs, the linter will make sure you replace them before moving between statuses.

Reply:
Do I need to replace these TBD before moving to a draft? I was wanting to get feedback/help from people on how to set them. I can just add best-guess variables if you think that would be better.

Reply:
No. You can have placeholders until Review.

It'd be even better to just not mention `FORK_BLOCK`. It isn't super important to your proposal, and is a holdover from ancient EIPs.